src/components/DpSearchField/DpSearchField.vue

```diff
@@ -7,13 +7,19 @@
       :input-attributes="{ placeholder: Translator.trans('search'), type: 'search' }"
       @reset="handleReset"
       @enter="handleSearch"
-      v-model="searchTerm" /><!--
+      v-model="searchTerm">
+      <!-- Slot for additional buttons -->
+      <slot />
+    </dp-resettable-input><!--
```

Comment:
I am pretty sure when placing a `inline-flex` on the outermost span, both the html comments and the `align-top` class could be dropped.

Reply:
Yes, that seems to work :tada: 9c3d56cfaca469a2833bc98cdeacf7ab9069fc6f